src/clojure/nrepl/core.clj

```diff
@@ -210,6 +210,10 @@
    such as HTTP, HTTPS, JMX, existing message queues, etc."
   uri-scheme)
 
+(defmulti transport->uri-scheme
```

Comment:
Btw, thinking about this a bit more - probably this should just go to the `transport` ns and be named `uri-scheme` there.